school_library_people.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = 1
+  end
+
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    @age > 18
```

Comment:
- [ ] This check is to verify if the age is greater or equal to 18. What you checked for was if the age is greater than 18.